modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/utils.js

```diff
@@ -22,15 +22,23 @@ function isIdDuplicated(elements, id) {
 }
 
 function getModalInfo(itemType) {
-	if (itemType === 'condition') {
+	if (itemType === 'assignments') {
+		return {
+			message: Liferay.Language.get(
+				'are-you-sure-you-want-to-delete-all-assignments-and-their-settings'
+			),
+			title: Liferay.Language.get('delete-assignments'),
+		};
+	}
+	else if (itemType === 'condition') {
```

Comment:
@cgmonte The `else if` is not really necessary here because we return inside of every `if`, but that's ok.